nailgun/entities.py

```diff
@@ -4038,7 +4039,7 @@ def errata(self, synchronous=True, **kwargs):
         return _handle_response(response, self._server_config, synchronous)
 
     def traces(self, synchronous=True, **kwargs):
-        """List services that need restarting on the host
+        """List services that need restarting for the host
```

Comment:
In this case, _on_ the host was OK (because that is where the services run). But its not important.